rocket/templates/rocket/profile.html

```diff
@@ -17,5 +17,6 @@
          <a href="http://127.0.0.1:8000/edit_profile/">edit account</a>
 <br>
          <a href="http://127.0.0.1:8000/change_password/">change password</a>
-
+<br>
+         <p><a href="http://127.0.0.1:8000/userprofile_edit/">Edit userprofile</a>(bio,phone,etc.)</p>
```

Comment:
Again here. This will work locally but when we will host it at some URL, this will still be linked to the localhost rather then the userprofile_edit of the website.